### ISSUE TITLE
help

### ISSUE THREAD

**Issue by zlx362211854** [kelseyhightower/nocode#2569]
Why can't I see the code on my computer? Is there a problem with my graphics card?
![Uploading image.png…]()

**Comment by thomas-de-tank** [comment]
Yes there is. Please try re-inserting the graphics card, after gently blasting it with an oxy-acetylene torch. After this it should be able to execute nocode.

**Comment by FrancoisRab** [comment]
I had the same issue. Thanks @thomas-de-tank.

**Comment by zlx362211854** [comment]
> Yes there is. Please try re-inserting the graphics card, after gently blasting it with an oxy-acetylene torch. After this it should be able to execute nocode.

I shot my computer with a firearm, as you said, and that's it，what should i do next
![boom](https://user-images.githubusercontent.com/22437181/47471061-43628400-d83b-11e8-9e22-a4977167811c.jpeg)

**Comment by Silicasandwhich** [comment]
Light your computer on fire and throw it into a barrel of oil. 

After that, reinstall your OS with a 1 TB swap partition

**Comment by fritx** [comment]
Stop it @Silicasandwhich
